.github/workflows/deploy.yml

```diff
@@ -13,7 +13,8 @@ jobs:
         make build
     - name: Post to Splunk SOAR API
       run: |
-        make deploy
+        # make deploy
+        echo "WARNING: Uncomment the line above to start deploying."
```

Comment:
I would like to see this change reverted.

Reply:
Let's take care of this in a dedicated change at #8

Reply:
Update: We were going to take care of this now, but need to work with Engineering to add it to the repo secrets.